wcomponents-core/src/test/java/com/github/bordertech/wcomponents/WMessageBox_Test.java

```diff
@@ -147,4 +148,12 @@ public void testTitleTextAccessors() {
 		resetContext();
 		Assert.assertNull("Default accessible text incorrect", comp.getTitleText());
 	}
+
+	@Test
+	public void testDuplicateComponentModels() {
+		WMessageBox messageBox = new WMessageBox(WMessageBox.Type.SUCCESS);
+		assertNoDuplicateComponentModels(messageBox, "type", WMessageBox.Type.INFO);
+//		No such method, invokeSetMethod doesn't like setting strings
+//		assertNoDuplicateComponentModels(messageBox, "titleText", "Test");
+	}
```

Comment:
@jonathanaustin  Just to drag this PR out even longer (lol) do you have any idea why invokeSetMethod won't work when setting Strings? Error message is Caused by: java.lang.NoSuchMethodException: Property 'titleText' has no setter method in class 'class com.github.bordertech.wcomponents.WMessageBox'. This happens on all the others with strings which is why I commented them out.